bin/start

```diff
@@ -34,14 +34,16 @@ function runScriptWithSummon() {
 }
 
 RUN_IN_DOCKER=false
-DEMO=false
 CONJUR_DEPLOYMENT=oss
 SUMMON_ENV=gke
+RELOAD_ENV=false
+DEV=false
 
 while true ; do
   case "$1" in
     --docker ) RUN_IN_DOCKER=true ; shift ;;
-    --demo ) DEMO=true ; shift ;;
+    --dev ) DEV=true ; shift ;;
+    --reload ) RELOAD_ENV=true ; shift ;;
```

Comment:
do you want to verify that the user didn't run `--reload` without `--dev`? raise something like:
```
if [[ $DEV = false && $RELOAD_ENV = true ]]; then
    echo "Error: --reload should be configured as part of --dev"
    exit 1
fi
```